examples/tailwind-css/src/index.tsx

```diff
@@ -9,11 +9,28 @@ import reportWebVitals from './reportWebVitals';
 const rootElement = document.getElementById('root');
 const root = createRoot(rootElement!);
 
+const theme = createTheme({
```

Comment:
```suggestion

// All `Portal`-related components need to have the the main app wrapper element as a container
// so that the are in the subtree under the element used in the `important` option of the Tailwind's config.
const theme = createTheme({
```
```suggestion
const theme = createTheme({
```